src/App.java

```diff
@@ -7,7 +7,8 @@ public static void main(String[] args) throws IOException {
                 "http://www.bungie.net/Platform/Destiny",
                 System.getenv("DESTINY_API_KEY")
         );
-        long membershipId = destinyService.getMembershipId(2, args[0]);
-        System.out.println(destinyService.getAccountSummary(2, membershipId));
+       long membershipId = destinyService.getMembershipId(2, "jonnyweb"); //args[0]
+       // long membershipId = destinyService.getMembershipId(2, "BeneficialBewick"); //args[0]
+        System.out.println(destinyService.getCharacterSummary(2, membershipId));
```

Comment:
I guess we should have a think about what we actually want to do here? Maybe let the user choose from several options and call the service based on that...?